transformation/src/index.ts

```diff
@@ -1,17 +1,42 @@
+import express, { Application } from 'express'
+import cors from 'cors'
+import bodyParser from 'body-parser'
+import session, { MemoryStore } from 'express-session'
+
+import Keycloak from 'keycloak-connect'
+
 import { TransformationEndpoint } from './transformationEndpoint'
 import VM2SandboxExecutor from './vm2SandboxExecutor'
 import JSTransformationService from './jsTransformationService'
 
+import { enableSwagger } from './swagger'
+
 const port = 8080
 
+const app: Application = express()
```

Comment:
Do we need that for swagger or did you just fix it on the fly?

Reply:
I did it because I wanted to declare the endpoint in a different file :D